dev/com.ibm.tx.jta/resources/com/ibm/ws/Transaction/resources/TransactionMsgs.nlsprops

```diff
@@ -870,5 +870,8 @@ WTRN0152_UNUSUAL_HA_CONFIG=WTRN0152W: Unusual HA configuration detected.
 WTRN0152_UNUSUAL_HA_CONFIG.explanation=HA is enabled for this cluster but the HA manager is disabled.
 WTRN0152_UNUSUAL_HA_CONFIG.useraction=If this is the desired configuration, no action is required. Otherwise, correct the HA configuration.
 
+WTRN0153_INVALID_LOG_AT_SHUTDOWN=WTRN0153W: The transaction service has shutdown but the recovery log had been marked as failed.
+WTRN0153_INVALID_LOG_AT_SHUTDOWN.explanation=The recovery log has failed and access to the log could not occur at server shutdown.
+WTRN0153_INVALID_LOG_AT_SHUTDOWN.useraction=The server must be restarted so that the log can be repaired. Try to correct any conditions indicated by the messaging that reported the invalidation of the log.
```

Comment:
```suggestion
WTRN0153_INVALID_LOG_AT_SHUTDOWN.useraction= Restart the server must be restarted so that the log can be repaired. Correct any conditions that are indicated by the messages that reported the invalidation of the log.
```

Not sure if user would need to correct the conditions before restarting the server to repair the log. If so, switch the order of the sentences